src/main/java/org/everit/jira/hr/admin/HolidaySchemesServlet.java

```diff
@@ -68,9 +68,9 @@
 
   private static final long serialVersionUID = 1073648466982165361L;
 
-  private final ManageSchemeComponent manageSchemeComponent =
-      new ManageSchemeComponent(this::listHolidaySchemes, this::saveScheme, this::updateScheme,
```

Comment:
Do you have 100 characters line length set in your eclipse formatter? It seems that there are formatting changes.